src/DotNetBridge/Bridge.cs

```diff
@@ -19,6 +21,8 @@
 using Microsoft.ML.Transforms;
 using Microsoft.ML.Transforms.TimeSeries;
 
+[assembly: LoadableClass(typeof(void), typeof(SchemaManipulation2), null, typeof(SignatureEntryPointModule), "SchemaManipulation2")]
```

Comment:
Should this be moved in to EntryPoints.cs?

Reply:
yep moved

---
In reply to: [326339191](https://github.com/microsoft/NimbusML/pull/270#discussion_r326339191) [](ancestors = 326339191)